src/services/onchain-data.service.ts

```diff
@@ -197,6 +198,26 @@ export class OnchainDataService implements OnModuleInit
         }
     }
 
+    private filterNonStandardPairsTokens(): void
+    {
+        for (const pairAddress in this.pairs)
+        {
+            // Filters out pairs where either one token is an LP token
+            if (this.pairs[pairAddress].token0.contractAddress in this.pairs
+              || this.pairs[pairAddress].token1.contractAddress in this.pairs)
```

Comment:
```suggestion
            // Filters out pairs where either one token is an LP token
            if (
                   this.pairs[pairAddress].token0.contractAddress in this.pairs
                || this.pairs[pairAddress].token1.contractAddress in this.pairs
               )
```

Reply:
ser what is the name of this latest fashion style?

Reply:
proxima style